go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/edgexfoundry/device-virtual-go
 
 require (
-	github.com/edgexfoundry/device-sdk-go v1.2.3
+	github.com/edgexfoundry/device-sdk-go v1.2.4-dev.12
```

Comment:
The upgrate to latest SDk should be in a separate PR from enabling DependaBot.

Reply:
removed dependabot from this PR

Reply:
@lenny-intel I addressed your change request. please help review again.  thanks.